### PR TITLE
Fixed notification issue in Facility Form

### LINKED ISSUE #3044: Facility Form : Malfunction
https://www.loom.com/share/21b64ffb4d48472cae80dff817a52924

even though the data is not getting saved, it is getting submited

### PR DESCRIPTION
**Pull request by cp-Coder**
# Updates

- [x] Fixes #3044
- [x] Fixed `notification` logic for `Facility Form`

### CODE REVIEW

**Review by skks1212 — APPROVED** [review]

File: src/Components/Facility/FacilityCreate.tsx [hunk 1]
```diff
@@ -456,19 +456,22 @@ export const FacilityCreate = (props: FacilityProps) => {
       const res = await dispatchAction(
         facilityId ? updateFacility(facilityId, data) : createFacility(data)
       );
+
       if (res && res.data) {
```
Comment (Ashesh3) [inline src/Components/Facility/FacilityCreate.tsx]: We can merge these two `if` statements.

```
if (res && res.data) {
    if (res.status === 200) {
```

```suggestion
      if (res && res.status === 200 && res.data) {
```

**Review by Ashesh3 — CHANGES_REQUESTED** [review]
Minor change, otherwise LGTM

**Comment by nihal467** [comment]
@cp-Coder make the notification more user friendly

**Comment by aeswibon** [comment]
@nihal467 This error is not present anymore.
If a user tries to assign `0: Choose Ward` as a ward, it will show `Field is required`. The form will not be saved.

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**